engine/gamesys/src/gamesys/scripts/script_collectionproxy.cpp

```diff
@@ -180,7 +186,8 @@ namespace dmGameSystem
 
     static const luaL_reg Module_methods[] =
     {
-        {"missing_resources", CollectionProxy_MissingResources},
+        {"missing_resources", Deprecated_CollectionProxy_GetMissingResources},
+        {"get_missing_resources", CollectionProxy_GetMissingResources},
```

Comment:
It felt wrong to add more "get_" functions, and at the same time leave this behind.